server/base/src/test/resources/accumulo.metrics.configuration.properties

```diff
@@ -25,6 +25,8 @@ metrics.service.02.name=PrometheusRegistry
 metrics.service.02.classname=org.apache.accumulo.server.metrics.service.prometheus.PrometheusMetricsRegistration
 metrics.service.02.enabled=true
 metrics.service.02.manager.PortNumber=9234
+metrics.service.02.gc.PortNumber=9235
+metrics.service.02.test.PortNumber=9236 
```

Comment:
There is an extra space after 9236 that causes the loadingTest to fail.

Reply:
Should be corrected as of 77212f9